doc/CONTRIBUTING.md

```diff
@@ -102,6 +102,20 @@ Good commit message subject :
 Document how to install the project
 ```
 
+Also, when you specify what you did avoid commit message subjects with "Fix bug
+in such and such feature". Saying you are fixing something implies the previous
+implementation was wrong and yours is right, which might not be even true.
+Instead, humbly say what you did technically and **then** explain how this is
```

Comment:
Yeah that was uncalled for, I'll rework that. Thanks for helping me finding the words.